R/g_lineplot.R

```diff
@@ -254,6 +255,7 @@ g_lineplot <- function(label = "Line Plot",
                        xlabel = xtick,
                        rotate_xlab = FALSE,
                        plot_font_size = 12,
+                       dot_size = 3,
```

Comment:
I was going to ask to assert numeric for this param. But, I don't see them done for any of the other params so I think it's okay to just leave it. Do you think we need it?

Reply:
yeah may be if there is refactor . we can make all the parameter consistent and add all the assert.